src/pkg/packager/prepare.go

```diff
@@ -211,7 +211,14 @@ func (p *Packager) FindImages() (imgMap map[string][]string, err error) {
 			for _, image := range sortedImages {
 				// Use print because we want this dumped to stdout
 				imagesMap[component.Name] = append(imagesMap[component.Name], image)
-				componentDefinition += fmt.Sprintf("      - %s\n", image)
+				imageWithSha, err := utils.GetImageWithSha(image)
+				if err != nil {
+					message.WarnErrf(err, "Problem getting image with sha for %s: %s", image, err.Error())
+					erroredCosignLookups = append(erroredCosignLookups, image)
```

Comment:
Cosign signatures are different than sha signatures. I recommend creating a new array for erroredShaLookups. You can place a message near erroredCosginLookups in the below block further down in the file

Reply:
added a check with erroredShaLookups